index.js

```diff
@@ -1,14 +1,15 @@
 const { Amplify } = require('aws-amplify')
 
-const Auth = require('./src/auth')
```

Comment:
musta been deep in that peyote when I did this

Reply:
no, you just copied it as it was in REACTor; I goofed it up originally.